### ISSUE TITLE
Remove `occt` version pin from conda install instructions?

### ISSUE THREAD

**Issue by whophil** [CadQuery/cadquery#1369]
The install instructions in the README and docs state:

```
conda install -c conda-forge cadquery occt=7.7.0
```

Given that `occt` is required and pinned by the [cadquery-feedstock](https://github.com/conda-forge/cadquery-feedstock/blob/1501e79249fa6eb45144733195690aaf046b4958/recipe/meta.yaml#L27-L28), I think it can be simplifed to
```
conda install -c conda-forge cadquery
```

**Comment by lorenzncode** [comment]
Agree, the instructions are now out of date.  The pin was required for version 2.2.

**Comment by adam-urbanczyk** [comment]
Solved in #1362
